pulp/build.gradle

```diff
@@ -28,10 +29,11 @@ android {
 }
 
 def deps = [
-        Room: 'androidx.room:room-runtime:2.0.0',
-        RoomAnnotation: 'androidx.room:room-compiler:2.0.0',
-        LifeCycleExtension: 'androidx.lifecycle:lifecycle-extensions:2.0.0',
-        LifeCycleAnnotation: 'androidx.lifecycle:lifecycle-compiler:2.0.0',
+        Room: 'androidx.room:room-runtime:2.2.5',
+        RoomRx: 'androidx.room:room-rxjava2:2.2.5',
```

Comment:
Why did you add `Room-Rx` when there's no use of that?
I suggest that you remove it because it highly increases the size and method count of the library.
Try to keep it as simple as possible.